apps/pagetour-sdk/src/models/navigatoroptions.ts

```diff
@@ -21,6 +21,8 @@ interface NavigatorOptions {
   callbackForTags?: () => Promise<Tags>
   callbackOnVolumeMute?: () => void
   callbackOnVolumeUnmute?: (transcript: string) => void
+  callbackOnPagetourFeedbackSubmit?: (rating: number, tour: Tutorial) => void
```

Comment:
see if we can club this and make stepno as optional parameter

Reply:
clubbed into callbackOnFeedbackSubmit